src/test/java/org/sagebionetworks/bridge/sdk/integration/SchedulePlanTest.java

```diff
@@ -135,6 +139,52 @@ public void crudSchedulePlan() throws Exception {
     }
 
     @Test
+    public void criteriaScheduleStrategyPlanCRUD() {
+        // Create plan with a criteria strategy
+        Schedule schedule1 = new Schedule();
+        schedule1.setLabel("Task 1");
+        schedule1.setScheduleType(ScheduleType.ONCE);
+        schedule1.addActivity(new Activity("Do task",null,new TaskReference("task:AAA")));
+        
+        Schedule schedule2 = new Schedule();
+        schedule2.setLabel("Task 2");
+        schedule2.setScheduleType(ScheduleType.ONCE);
+        schedule2.addActivity(new Activity("Do task",null,new TaskReference("task:BBB")));
+        
+        ScheduleCriteria criteria1 = new ScheduleCriteria.Builder()
+                .withMinAppVersion(2)
+                .withMaxAppVersion(5)
+                .withSchedule(schedule1).build();
+        ScheduleCriteria criteria2 = new ScheduleCriteria.Builder()
+                .withMinAppVersion(6)
+                .withMaxAppVersion(10)
+                .withSchedule(schedule2).build();
+        
+        CriteriaScheduleStrategy strategy = new CriteriaScheduleStrategy();
+        strategy.addCriteria(criteria1);
+        strategy.addCriteria(criteria2);
+        
+        SchedulePlan plan = new SchedulePlan();
+        plan.setLabel("Criteria schedule plan");
+        plan.setStrategy(strategy);
+        
+        GuidVersionHolder keys = developerClient.createSchedulePlan(plan);
+        SchedulePlan retrievedPlan = developerClient.getSchedulePlan(keys.getGuid());
+        
+        assertTrue(retrievedPlan.getStrategy() instanceof CriteriaScheduleStrategy);
+        
+        CriteriaScheduleStrategy retrievedStrategy = (CriteriaScheduleStrategy)retrievedPlan.getStrategy();
+        assertEquals(2, retrievedStrategy.getScheduleCriteria().size());
```

Comment:
Changed the next two lines to compare with the original criteria objects... they really have to be the same to be equal. Requires passing GUID generated on server back to original object for them to be truly equal; added utility method.